.github/workflows/dev-mode-layer-publish.yml

```diff
@@ -0,0 +1,51 @@
+name: "Dev Mode Extension: Publish"
+on:
+  push:
+    tags:
+      - "@serverless/dev-mode-extension@[0-9]+.[0-9]+.[0-9]+"
+defaults:
+  run:
+    working-directory: dev-mode
+
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16
+          registry-url: "https://npm.pkg.github.com"
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '>=1.18.0'
+      - name: Build layer
+        run: make build
+      - name: Publish layers (dev)
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.OPEN_DEV_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.OPEN_DEV_AWS_SECRET_ACCESS_KEY }}
+        run: |
+          TEMP_ARRAY=($(echo $GITHUB_REF | tr "@" "\n"))
+          VERSION=${TEMP_ARRAY[@]: -1}
+          ./scripts/publish-extension-layers.js \
```

Comment:
For that to work I think we need to traverse into `node` directory, and also ensure all dependencies there are installed

Reply:
Yes 🤦‍♂️ when I was double checking some stuff I noticed this too I fixed this in another commit.